src/test/java/com/okta/tools/aws/settings/CredentialsTest.java

```diff
@@ -35,19 +34,10 @@
     private String accessKey = "accesskey";
     private String secretKey = "secretkey";
     private String sessionToken = "sessiontoken";
-    private String manualRole = "[" + roleName + "_source]\n"
+    private String manualRole = "[" + roleName + "]\n"
             + Credentials.ACCES_KEY_ID + " = " + accessKey + "\n"
             + Credentials.SECRET_ACCESS_KEY + " = " + secretKey + "\n"
             + Credentials.SESSION_TOKEN + " = " + sessionToken;
-    private String manualRoleCustomSuffix = "[" + roleName + "_custom]\n"
-            + Credentials.ACCES_KEY_ID + " = " + accessKey + "\n"
-            + Credentials.SECRET_ACCESS_KEY + " = " + secretKey + "\n"
-            + Credentials.SESSION_TOKEN + " = " + sessionToken;
-    private OktaAwsCliEnvironment environmentWithCustomSuffix =
-            new OktaAwsCliEnvironment(false, null, null,
-                    null, null, null, null,
-                    null, 0, null,
-                    null, false, "_custom");
```

Comment:
:fire: suffix customization was a workaround for the workaround causing problems for people using this with downstream tools. Removing the writes to ~/.aws/config makes suffix customization unnecessary.